### ISSUE TITLE
 'java' is not recognized as an internal or external command,

### ISSUE THREAD

**Issue by GaryXiongxiong** [valentjn/vscode-ltex#75]
Hi,
I am trying to install this extension. But it doesn't work.

I tried to use my own JDK path or the path of JDK downloaded automatically, it always says that 'java' is not recognized as an internal or external command.

I also tried to run ltex-ls in command line, and it works.
```
❯ .\ltex-ls.bat --version
{
  "ltex-ls": "6.0.1",
  "java": "11.0.7"
}
```

 I am wondering why it could not recognize 'java' during its execution.

Thanks a lot

```
2020-07-11T12:57:41.163Z Info: Testing ltex-ls...
2020-07-11T12:57:41.163Z Info:   Command: "c:\\Users\\Yixio\\.vscode\\extensions\\valentjn.vscode-ltex-6.0.1\\lib\\ltex-ls-6.0.1\\bin\\ltex-ls.bat"
2020-07-11T12:57:41.163Z Info:   Arguments: ["--version"]
2020-07-11T12:57:41.163Z Info:   env['JAVA_HOME']: "c:\\Users\\Yixio\\.vscode\\extensions\\valentjn.vscode-ltex-6.0.1\\lib\\jdk-11.0.7+10-jre"
2020-07-11T12:57:41.163Z Info:   env['JAVA_OPTS']: "-Xms64m -Xmx512m"
2020-07-11T12:57:41.199Z Info: Test failed.
2020-07-11T12:57:41.199Z Info: Exit code of ltex-ls: 9009
2020-07-11T12:57:41.199Z Info: stdout of ltex-ls:
2020-07-11T12:57:41.199Z Info: 
2020-07-11T12:57:41.199Z Info: stderr of ltex-ls:
2020-07-11T12:57:41.199Z Info: 'java' is not recognized as an internal or external command,
2020-07-11T12:57:41.199Z Info: operable program or batch file.
2020-07-11T12:57:41.199Z Info: 
2020-07-11T12:57:41.200Z Error: The download/extraction/run of Java failed!
2020-07-11T12:57:41.200Z Error: Error details:
2020-07-11T12:57:41.200Z Error: Error: Could not run ltex-ls.
2020-07-11T12:57:41.200Z Error: 	at w.<anonymous> (c:\Users\Yixio\.vscode\extensions\valentjn.vscode-ltex-6.0.1\dist\extension.js:1:290092)
2020-07-11T12:57:41.200Z Error: 	at Generator.next (<anonymous>)
2020-07-11T12:57:41.200Z Error: 	at o (c:\Users\Yixio\.vscode\extensions\valentjn.vscode-ltex-6.0.1\dist\extension.js:1:281234)
```

**Comment by valentjn** [comment]
This is a regression from 6.0.0: With the migration from Gradle to Maven, LT<sub>E</sub>X LS migrated the startup scripts from being generated by the [Application Gradle plugin](https://docs.gradle.org/current/userguide/application_plugin.html) to the [AppAssembler Maven plugin](https://www.mojohaus.org/appassembler/appassembler-maven-plugin/). The latter [doesn't take `JAVA_HOME` into account under Windows](https://github.com/mojohaus/appassembler/blob/appassembler-2.1.0/appassembler-maven-plugin/src/main/resources/org/codehaus/mojo/appassembler/daemon/script/windowsBinTemplate), unlike under Unix-like OSs, where [it does](https://github.com/mojohaus/appassembler/blob/appassembler-2.1.0/appassembler-maven-plugin/src/main/resources/org/codehaus/mojo/appassembler/daemon/script/unixBinTemplate).

**Comment by valentjn** [comment]
Fix released in 6.0.2.
